api/_globals/createImageBitmap.json

```diff
@@ -109,6 +109,78 @@
           }
         }
       },
+      "options_imageOrientation_parameter_from-image_value": {
+        "__compat": {
+          "description": "<code>options.imageOrientation</code> parameter value <code>from-image</code>",
+          "support": {
+            "chrome": {
+              "version_added": "112"
+            },
+            "chrome_android": "mirror",
+            "deno": {
+              "version_added": "1.40"
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "111"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "16"
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "options_imageOrientation_parameter_none_value": {
+        "__compat": {
+          "description": "<code>options.imageOrientation</code> parameter value <code>none</code>",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "deno": {
+              "version_added": "1.40"
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
```

Comment:
Note, this is technically not "standard" yet as it isn't in the spec. But deno has it, and the intent is to do this. I think they are just creating space from `none` having a meaning on other platforms.

Reply:
Let's mark `standard_track` as `false` then, because the definition of `standard_track` is that the feature is present in a current specification.